lib/measured/unit_system_builder.rb

```diff
@@ -8,12 +8,56 @@ def unit(unit_name, aliases: [], value: nil)
     nil
   end
 
+  def si_unit(unit_name, aliases: [], value: nil)
+    @units += build_si_units(unit_name, aliases: aliases, value: value)
+    nil
+  end
+
   def build
     Measured::UnitSystem.new(@units)
   end
 
   private
 
+  def self.prefixes
+    [
+      ['y', 'yocto', -24],
+      ['z', 'zepto', -21],
+      ['a', 'atto', -18],
+      ['f', 'femto', -15],
+      ['p', 'pico', -12],
+      ['n', 'nano', -9],
+      ['μ', 'micro', -6],
+      ['m', 'milli', -3],
+      ['c', 'centi', -2],
+      ['d', 'deci', -1],
+      ['', '', 0],
+      ['da', 'deca', 1],
+      ['h', 'hecto', 2],
+      ['k', 'kilo', 3],
+      ['M', 'mega', 6],
+      ['G', 'giga', 9],
+      ['T', 'tera', 12],
+      ['P', 'peta', 15],
+      ['E', 'exa', 18],
+      ['Z', 'zetta', 21],
+      ['Y', 'yotta', 24]
+    ]
+  end
+
+  def build_si_units(name, aliases: [], value: nil)
+    value = "1 #{name}" if !value
+    size, unit = value.split(' ')
+    size ||= 1
```

Comment:
I'm not sure if this is the best way to do this, it seems sketchy, as it will fail if someone sets the value to be ```1000 weight ton``` as there are 2 spaces.